cfgov/unprocessed/css/organisms/mega-menu.less

```diff
@@ -295,23 +295,26 @@ body {
         // Menu flyout.
         // All menus - Tablet/Mobile.
         &_content {
+            .u-drop-shadow-after();
```

Comment:
Where is this showing up?

Reply:
IDK, but it was already there, I just moved it to the top because that's our rule for mixins.

Reply:
<img width="399" alt="screen shot 2018-06-25 at 2 29 22 pm" src="https://user-images.githubusercontent.com/1280430/41871311-2ad6d286-7884-11e8-8365-4615ca858116.png">

It appears you just can't see it due to the dark overlay on the rest of the page.